components/Navigation/Navigation.js

```diff
@@ -0,0 +1,42 @@
+import {
+  Nav,
+  Footer,
+  NavLink,
+  MapIcon,
+  StarIcon,
+  BikeIcon,
+  RouteIcon,
+} from "./StyledNavigation";
+
+export default function Navigation({ activePage }) {
+  return (
+    <Footer>
+      <Nav>
+        <NavLink isactive={activePage === "/" ? "on" : "off"} href="/">
+          <MapIcon />
+        </NavLink>
+
+        <NavLink
+          isactive={activePage === "/routes" ? "on" : "off"}
+          href="/routes"
+        >
+          <RouteIcon />
+        </NavLink>
+
+        <NavLink
+          isactive={activePage === "/routes/favorites" ? "on" : "off"}
+          href="/routes/favorites"
+        >
+          <StarIcon />
+        </NavLink>
+
+        <NavLink
+          isactive={activePage === "/profile" ? "on" : "off"}
```

Comment:
using `"on"` and `"off"` works, but it is very strange to not use a boolean for a prop called `isactive`.

Also please consider using camelCase here.

```suggestion
          isActive={activePage === "/profile"}
```